### PR TITLE
[metricsadvisor] get pylint clean

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py [hunk 1]
```diff
@@ -612,7 +621,7 @@ def list_metric_dimension_values(self, metric_id, dimension_name, **kwargs):
             **kwargs)
 
     @distributed_trace
-    def list_metrics_series_data(self, metric_id, start_time, end_time, filter, **kwargs):
+    def list_metrics_series_data(self, metric_id, start_time, end_time, series_to_filter, **kwargs):
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py:624]: original parameter name `filter` was overshadowing the built-in filter. Other languages call this parameter `series_to_filter` so I renamed it.

**Review by xiangyan99 — DISMISSED** [review]

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**